### DISCUSSION TITLE
Configuring the default dashboard with separate Powerwall 2 and Inverters

### DISCUSSION THREAD

**Discussion by markm68k** [jasonacox/Powerwall-Dashboard#327] [Q&A] [answered]
The default dashboard is configured for Powerwall+ devices, but I have two Powerwall 2 devices, two separate Inverters, and a separate Gateway.

How can I configure my dashboard to show my setup?

See the attached screenshot for details.

<img width="1169" alt="image" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/45903780/eccfa443-e153-4981-8db4-ddeff81cc3ea">

**Comment by anderep** [comment, marked as answer]
Tesla doesn’t give us access to inverters, only pw2+. I have enphase and tesla solar inverter, I was able to mine the data from each system thru neurio data.

**Reply by jasonacox** [reply]
Sadly, that's true. @anderep any tips on getting that neurio data? :)

**Reply by markm68k** [reply]
Thanks! I figured that was the answer.

**Reply by anderep** [reply]
Look at this thread for mining data from neurio…
https://github.com/jasonacox/Powerwall-Dashboard/discussions/254
